File: c/bee1011.c
#include <stdio.h>
#include <math.h>
 
int main() {
    double pi, raio, volume;
    pi = 3.14159;
    scanf("%lf", &raio);
    volume = (4/3.0) * pi * pow(raio,3);

    printf("VOLUME = %.3lf\n", volume);
    return 0;
}

File: java/bee1011.java
import java.util.Scanner;

public class bee1011 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int ray = input.nextInt();
        double volume = getVolume(ray);

        System.out.printf("VOLUME = %.3f\n", volume);

        input.close();
    }

    public static double getVolume(int ray) {
        return (4/3.0) * 3.14159 * Math.pow(ray, 3);
    }
}
